tailormade_dls/src/main/res/layout/layout_chat_content_reply.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:background="@drawable/drawable_chat_reply"
+    android:elevation="@dimen/dp_4"
+    android:orientation="horizontal">
+
+    <TextView
+        android:id="@+id/text_view_chat_content_reply_text"
+        style="?attr/textAppearanceBody2"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginHorizontal="@dimen/dp_12"
+        android:layout_marginVertical="@dimen/dp_8"
+        android:text="@string/chat_content_label"
+        android:textColor="@color/color_black_87" />
+
+    <ImageView
+        android:id="@+id/image_view_chat_content_reply"
+        android:layout_width="@dimen/dp_50"
+        android:layout_height="@dimen/dp_50"
+        android:layout_marginHorizontal="@dimen/dp_12"
+        android:layout_marginVertical="@dimen/dp_8"
+        android:visibility="gone"
```

Comment:
suggestion if needed, all visibility gone must be followed by tools:visibility so it can visible in preview